src/beamlime/config/raw_detectors/dream.py

```diff
@@ -2,41 +2,40 @@
 # Copyright (c) 2024 Scipp contributors (https://github.com/scipp)
 
 # Order in 'resolution' matters so plots have X as horizontal axis and Y as vertical.
-# The other DREAM detectors have non-consecutive detector numbers. This is not
-# supported currently
 
 import scipp as sc
 from ess.reduce.live import raw
 
-_res_scale = 12
+_res_scale = 8
+pixel_noise = sc.scalar(4.0, unit='mm')
 
 dream_detectors_config = {
-    'dashboard': {'nrow': 3, 'ncol': 2},
```

Comment:
This and `gridspec` below were used in the prototype last autumn but are no longer in use.